src/management-system-v2/lib/data/processes.tsx

```diff
@@ -118,6 +140,34 @@ export const addProcesses = async (
   return newProcesses;
 };
 
+export const updateProcessMetaInfo = async (
```

Comment:
With this name i would expect this function to be usable to change arbitrary entries of  a process' meta data but it only accepts values for sharing related entries.